core/modules/contextual/js/contextual.js

```diff
@@ -16,30 +16,107 @@ Backdrop.behaviors.contextualLinks = {
       var $wrapper = $(this);
       var $region = $wrapper.closest('.contextual-links-region');
       var $links = $wrapper.find('ul.contextual-links');
-      var $trigger = $('<a class="contextual-links-trigger" href="#" />').text(Backdrop.t('Configure')).on('click',
+      var $trigger = $('<a class="contextual-links-trigger" href="#" />').text(Backdrop.t('Configure')).click(
         function () {
           $links.stop(true, true).slideToggle(100);
           $wrapper.toggleClass('contextual-links-active');
           return false;
         }
       );
       // Attach hover behavior to trigger and ul.contextual-links.
-      $trigger.add($links).on('hover',
+      $trigger.add($links).hover(
```

Comment:
The hover problem is being addressed separately in https://github.com/backdrop/backdrop-issues/issues/6223. We'll need to revert these after that PR is merged.

Reply:
Yes, I saw the changes and will do the rebase against 1.x after it's merged.

Reply:
New code incorporated.